src/analysis/timeline/timelineCredView.js

```diff
@@ -0,0 +1,99 @@
+// @flow
+
+import {sum} from "d3-array";
+import sortBy from "lodash.sortby";
+import stringify from "json-stable-stringify";
+import * as NullUtil from "../../util/null";
+import {TimelineCred, type CredNode} from "./timelineCred";
```

Comment:
We should move the `CredNode` type into this module, since now the `timelineCred.js` module does not create or return any `CredNode`s.